src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFilterTypes.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using System.Text;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement
+{
+    /// <summary>
+    /// Types of built-in feature filters.
+    /// </summary>
+    internal class FeatureFilterTypes
+    {
+        private const string CustomFilter = "Custom";
+        private const string PercentageFilter = "Percentage";
+        private const string TimeWindowFilter = "Time";
```

Comment:
I'd stick with the full name, TimeWindow.

Reply:
I'd say let's use shorter names for all. How about these - Percent, Time, Target ?

Reply:
Why not enum in that case?

Reply:
If we want to save characters I feel we need to go all the way.

Thats like this

`RequestType=Watch,Filter=0+1+2+3`

Reply:
If we use codes, we need to remember what each code number means while looking at logs. Our PowerBI query will also need to convert the codes to names, so anyone modifying the query needs to check the source code to know what each code means. Having somewhat trimmed names provide a balance between saving space and maintaining human readability.

Reply:
These names aren't made up, Time could be a valid filter. TimeWindow, is our filter.

Reply:
I don't feel great about squeezing characters halfway. If there is a concern of correlation context length, we need to consider real solution which is shortening keys and values to enum where appropriate. If not, I don't see any value to introduce drift from our real names.

Reply:
How do you feel about using codes like this:
```
private enum FilterType
{
    [Description("CSTM")]
    Custom,
    [Description("PRCNT")]
    Percentage,
    [Description("TIME")]
    TimeWindow,
    [Description("TRGT")]
    Targeting
}
```

So the header looks like this:
`RequestType=Watch,Filter=CSTM+PRCNT+TIME+TRGT`

Reply:
I'm fine with that, it's a consistent approach with that method. cc @zhenlan to see if he has any concern about it.

I would stick away from defining an enum and using attribute to get those strings though. Private method where we are generating the correlation context should be fine.